framework/wazuh/core/batcher/client.py

```diff
@@ -23,40 +21,26 @@ def __init__(self, queue: MuxDemuxQueue, wait_frequency: float = 0.1):
         self.queue = queue
         self.wait_frequency = wait_frequency
 
-    def send_event(self, agent_metadata: AgentMetadata, event: StatefulEvent) -> int:
+    def send_event(self, agent_metadata: AgentMetadata, header: Header, event: StatefulEvent = None):
```

Comment:
Since the DELETE operation does not require an event to be sent, we could rename the method:
```suggestion
    def send_operation(self, agent_metadata: AgentMetadata, header: Header, event: StatefulEvent = None):
```